packages/plugin-browserify/package.json

```diff
@@ -50,13 +50,13 @@
     "eslint-plugin-jsdoc": "^36.1.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.4.0",
-    "jest": "^27.5.1",
+    "jest": "^29.0.2",
```

Comment:
This means we have two versions of Jest in the repo right? Is that intentional? IIRC there is some configuration to do before we can upgrade jest everywhere so it might be out of scope for this.